packages/components/src/tabs/styles.ts

```diff
@@ -13,12 +13,35 @@ import { space } from '../utils/space';
 import { reduceMotion } from '../utils/reduce-motion';
 
 export const TabListWrapper = styled.div`
+	position: relative;
 	display: flex;
 	align-items: stretch;
 	flex-direction: row;
 	&[aria-orientation='vertical'] {
 		flex-direction: column;
 	}
+	&.isAnimationEnabled::after {
```

Comment:
nit: the CSS class naming convention used across Gutenberg is more like `.is-animation-enabled`.

Reply:
More info about our BEM dialect in the [contributing guidelines](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/code/coding-guidelines.md).

Reply:
This is a conscious decision I made for authoring components a long time ago, after getting annoyed with this kind of thing:

```ts
<div className={clsx({
  "is-animation-enabled": isAnimationEnabled,
  "is-whatever": isWhatever,
  "is-this": isThis,
  "is-that": isThat,
})} />
```

Doing camel case unlocks this pattern:

```ts
<div className={clsx({
  isAnimationEnabled,
  isWhatever,
  isThis,
  isThat,
})} />
```

That said, I agree I should be following the convention. I might propose this as a broader change in the future though.

Reply:
The CSS class naming conventions are inherited from (and compatible with) the [WordPress CSS coding standards](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/css/#selectors). Changing those at this time is a near-impossible task.